src/components/Squeak/components/EditProfile.tsx

```diff
@@ -82,8 +83,9 @@ function Avatar({ values, setFieldValue }) {
 
     useEffect(() => {
         if (!values.avatar && inputRef?.current) {
-            inputRef.current.value = null
+            inputRef.current.value = ''
```

Comment:
I believe the value returned here needs to be null in order for line 175 to work properly. Feel free to refactor to avoid the type errors this throws.

Reply:
I'll just leave this the way it was for now, not a big deal and can be fixed later 👍